src/region.js

```diff
@@ -269,7 +275,11 @@ const Region = MarionetteObject.extend({
     delete this.currentView;
 
     if (!view._isDestroyed) {
-      this._removeView(view, shouldDestroy);
+      if (shouldDestroy) {
+        this.removeView(view);
+      } else {
+        this._detachView(view);
```

Comment:
so in the codepen `mainView.showChildView('content', view, { preventDestroy: true });` messes things up a bit.  Would that same thing be true with this implementation?